Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:9
+
+RUN mkdir -p /usr/src
+WORKDIR /usr/src/
+
+ARG NODE_ENV
```

Comment:
What is this used for? my reading suggests it modifies the FROM image directive? https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
Yeah, this was leftover from the SW Dockerfile - I've taken it out since it's set to `production` further down.